packages/webapp/src/pages/Team/components/Users.tsx

```diff
@@ -13,6 +13,10 @@ export const columns: ColumnDef<ApiUser | ApiInvitation>[] = [
         header: 'Name',
         size: 180,
         cell: ({ row }) => {
+            if ('created_at' in row.original) {
+                return '-';
+            }
+
```

Comment:
what is this for?

Reply:
when we invite there is no name available, I was putting the email but we decided it was clearer like this 
<img width="797" alt="Screenshot 2024-07-19 at 16 04 12" src="https://github.com/user-attachments/assets/18d7fd78-6efc-47a3-9c87-e688fd791372">